config/project.sh

```diff
@@ -4,12 +4,13 @@ export PROJECT_VERSION="0.0.1"
 
 export GOLANG_VERSION="1.9.2"
 
-export API_URL="dev.api.daniel-ortega.mx"
+export API_URL="dev.api.culturacolectiva.com"
 export API_PORT="3000"
 
-export DB_URL="dev.db.daniel-ortega.mx"
-export DB_PORT="3306"
-export DB_NAME="do_api"
+export DB_URL="dev.db.culturacolectiva.com"
+
+export DB_PORT="27017"
```

Comment:
This port is only for `mongo` instances, if it's here, the `gocker` tool as git submodule works as expected?

Reply:
the same comment above, I'll avoid this problem when the init module be ready